content/en/docs/concepts/workloads/pods/_index.md

```diff
@@ -15,11 +15,12 @@ card:
 _Pods_ are the smallest deployable units of computing that you can create and manage in Kubernetes.
 
 A _Pod_ (as in a pod of whales or pea pod) is a group of one or more
-{{< glossary_tooltip text="containers" term_id="container" >}}, with shared storage and network resources, and a specification for how to run the containers. A Pod's contents are always co-located and
+{{< glossary_tooltip text="containers" term_id="containers" >}}, that shares storage and network resources, 
```

Comment:
```suggestion
{{< glossary_tooltip text="containers" term_id="container" >}}, that shares storage and network resources, 
```
Do not modify the term_id, which may cause the build failure.

Reply:
corrected the term_id.